build-system/tasks/runtime-test/helpers.js

```diff
@@ -373,10 +431,10 @@ async function createKarmaServer(
   });
 
   karmaServer
-    .on('run_start', () => karmaRunStart())
-    .on('browsers_ready', () => karmaBrowsersReady())
-    .on('browser_start', browser => karmaBrowserStart(browser))
-    .on('browser_complete', browser => karmaBrowserComplete(browser))
+    .on('run_start', () => karmaRunStart_())
```

Comment:
While you're cleaning up this code, would this abbreviated notation work? (Will `browser` be implicitly passed in to `karmaBrowserStart_` and `karmaBrowserComplete_`?
```js
  karmaServer
    .on('run_start', karmaRunStart_)
    .on('browsers_ready', karmaBrowsersReady_)
    .on('browser_start', karmaBrowserStart_)
    .on('browser_complete', karmaBrowserComplete_)
    .on('run_complete', runCompleteFn);
```

Reply:
Done